bindings/react/src/components/Toolbar.jsx

```diff
@@ -33,6 +34,20 @@ class Toolbar extends SimpleWrapper {
   _getDomNodeName() {
     return 'ons-toolbar';
   }
+
+  componentDidMount() {
+    super.componentDidMount();
+
+    if (this.props.visible !== undefined) {
+      ReactDOM.findDOMNode(this).setVisibility(this.props.visible);
+    }
+  }
+
+  componentWillReceiveProps(nextProps) {
```

Comment:
I just found out `componentWillReceiveProps` is going to get deprecated and renamed to `UNSAFE_componentWillReceiveProps` in React 17. It's fine for React 16 though.

We use this function all over the place so I've created an [issue](https://github.com/OnsenUI/OnsenUI/issues/2482) so we can deal with it all at once.